map/index.js

```diff
@@ -1,8 +1,10 @@
 function multiplyBy10(array) {
-  for (let i = 0; i < array.length; i++) {
-    array[i] = array[i] * 10;
-  }
-  return array;
+ return array.map(x => x * 10);
+
+  // for (let i = 0; i < array.length; i++) {
+  //   array[i] = array[i] * 10;
+  // }
+  // return array;
 }
 
 function onlyVowels(array) {
```

Comment:
Finish these :) it is super important that you at least understand how map and filter functions work and be refactored into your code!
``` 
function onlyVowels(array) {
return array.map(word => word.replace(/[^aeiou]*/gi, ''));
}
```